addons/ofxiOS/src/video/AVFoundationVideoPlayer.m

```diff
@@ -326,7 +326,7 @@ - (BOOL)createAssetReaderWithTimeRange:(CMTimeRange)timeRange {
     }
     
     //------------------------------------------------------------ add audio output.
-    double preferredHardwareSampleRate = [[AVAudioSession sharedInstance] currentHardwareSampleRate];
+    double preferredHardwareSampleRate = [[AVAudioSession sharedInstance] sampleRate];
```

Comment:
issue for moving up ofxiOS to 6.0 is here #4431
@danoli3 is it possible to take elements from this PR to address this issue in a separate PR?